src/DearImGui/Raw.hs

```diff
@@ -462,6 +463,16 @@ arrowButton :: (MonadIO m) => CString -> ImGuiDir -> m Bool
 arrowButton strIdPtr dir = liftIO do
   (0 /=) <$> [C.exp| bool { ArrowButton($(char* strIdPtr), $(ImGuiDir dir)) } |]
 
+-- | Image Area to draw a texture
+--
+-- Wraps @ImGui::Image()
+-- See https://github.com/ocornut/imgui/wiki/Image-Loading-and-Displaying-Examples#About-texture-coordinates for an example
+-- For OpenGL: The Ptr points to the texture memory (eg. 0x0x0000000000000001), so it is the number from glBindTexture.
+-- Convert it directly with: intPtrToPtr . fromIntegral
+image :: (MonadIO m) => Ptr ()-> CFloat -> CFloat -> m()
```

Comment:
Please add remaining arguments to raw binding and use `Ptr ImVecN` on haskell side.

(args, for the reference)
```c++
Image(
  ImTextureID user_texture_id,
  const ImVec2& size,
  const ImVec2& uv0 = ImVec2(0,0), // texture region offset
  const ImVec2& uv1 = ImVec2(1,1), // texture region size
  const ImVec4& tint_col = ImVec4(1,1,1,1),
  const ImVec4& border_col = ImVec4(0,0,0,0)
);
```

Reply:
Thanks for your feedback!

> (...) and use Ptr ImVecN on haskell side.

Oh, I this is new to me! So, instead of copying values for variable references in the function I also have to pass a pointer?
(Calling `f(int *a)` from Haskell is the same as calling `f(int &a) `?)